cpp/src/transform/one_hot_encode.cu

```diff
@@ -62,12 +62,12 @@ std::pair<std::unique_ptr<column>, table_view> one_hot_encode(column_view const&
 {
   CUDF_EXPECTS(input.type() == categories.type(), "Mismatch type between input and categories.");
 
-  if (categories.is_empty()) { return std::pair(make_empty_column(type_id::BOOL8), table_view{}); }
+  if (categories.is_empty()) { return {make_empty_column(type_id::BOOL8), table_view{}}; }
```

Comment:
For future PRs, let's keep the unrelated refactoring separate from stream changes.

Reply:
Ah yeah sorry about that. I don't recall why I did this in this PR, I think it might have been an autofix from my LSP.

Reply:
I was about to applaud fixing this as a matter of course. :] Noted.

Reply:
@mythrocks I like the change, just not in these API changing PRs. The public stream PRs are going to be large and touch lots of files, so it’s best to keep their scope tight.